samples/gradle.properties

```diff
@@ -1,3 +1,6 @@
+avito.logging.verbosity=DEBUG
+avito.build=local
```

Comment:
@sboishtyan That was the reason for "teamcity" deployment label. new project and i forgot to set this flag. We even have TODO message about local by default